lib/server/formations.ts

```diff
@@ -189,3 +189,78 @@ export async function searchFormations(
 
   return formations;
 }
+
+async function _mostPopularFormations(limit: number): Promise<FormationData[]> {
```

Comment:
I trust you that the SQL works, that's not my strong suit 😅
Do you think there's a use case for being able to change the limit or would just fixing it, to 5 as used in Search, for now be more appropriate?

Reply:
I left it as an arg in case we want to expand the result set in another view like a landing page or a “most popular” page in the future